### PR TITLE
Add Cricut Design Space Plugin 5.8.1808.282223

### PR DESCRIPTION
**Pull request by wastrachan**
After making all changes to the cask:

- [x] `brew cask audit --download {{cask_file}}` is error-free.
- [x] `brew cask style --fix {{cask_file}}` reports no offenses.
- [x] The commit message includes the cask’s name and version.
- [x] The submission is for [a stable version](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#finding-a-home-for-your-cask) or [documented exception](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#but-there-is-no-stable-version).

Additionally, if **adding a new cask**:

- [x] Named the cask according to the [token reference].
- [x] `brew cask install {{cask_file}}` worked successfully.
- [x] `brew cask uninstall {{cask_file}}` worked successfully.
- [x] Checked there are no [open pull requests] for the same cask.
- [x] Checked the cask was not [already refused].
- [x] Checked the cask is submitted to [the correct repo].

[token reference]: https://github.com/Homebrew/homebrew-cask/blob/master/doc/cask_language_reference/token_reference.md
[open pull requests]: https://github.com/Homebrew/homebrew-cask/pulls
[already refused]: https://github.com/Homebrew/homebrew-cask/search?q=is%3Aclosed&type=Issues
[the correct repo]: https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#finding-a-home-for-your-cask
[version-checksum]: https://github.com/Homebrew/homebrew-cask/blob/master/doc/cask_language_reference/stanzas/sha256.md#updating-the-sha256

### CODE REVIEW

File: Casks/cricut-design-space.rb [hunk 1]
```diff
@@ -0,0 +1,20 @@
+cask 'cricut-design-space' do
+  version '5.8.1808.282223'
+  sha256 '048483d69974d24bd99880bcc3c6cf51643c8a53728351cdf31ec476c625f38b'
+
+  url "https://imgservice.cricut.com/design-public-mirror1/software/Plugin/CricutDesignSpace-#{version}.zip"
+  name 'Cricut Design Space Plugin'
+  homepage 'https://cricut.com/'
+
+  installer script: {
+                      executable: "#{staged_path}/CricutDesignSpace-#{version}.app/Contents/MacOS/installbuilder.sh",
```
Comment (vitorgalvao) [inline Casks/cricut-design-space.rb]: Isn’t there something similar to uninstall?

File: Casks/cricut-design-space.rb [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+                      args:       ['--mode', 'unattended', '--unattendedmodeui', 'none'],
+                    }
+
+  uninstall trash:      '/Applications/Cricut Design Space.app',
+            quit:       [
+                          'com.ProvoCraft.CricutBridge4',
```
Comment (vitorgalvao) [inline Casks/cricut-design-space.rb]: You sure it’s `4`?

Comment (wastrachan) [reply Casks/cricut-design-space.rb]: It's definitely 4, although it does not make a lot of sense. 

```
>> homebrew-cask/developer/bin/list_running_app_ids | grep Cricut

com.ProvoCraft.CricutBridge4	CricutBridge4
com.ProvoCraft.CricutLauncher4	CricutLauncher4
```

File: Casks/cricut-design-space.rb (hunk 1 above)
Comment (wastrachan) [reply Casks/cricut-design-space.rb]: Not that I could find, and their official instruction for uninstallation is the "trash drag". I based the uninstall stanza on these two factors. 

image

Nothing exists in the installed app, and the installbuilder script included in the installer does not accept any uninstallation flags.

**Pull request merged**